src/i18n/Localizable.xcstrings

```diff
@@ -20000,6 +20105,7 @@
       }
     },
     "Retry" : {
+      "comment" : "is this one used as verb or noun?",
```

Comment:
Verb => Action to retry something

Reply:
changed noun to gerund. same reason for"Restart", `because in Korean noun or gerund is usually used to implement actions`